src/components/html/EcApplication.html

```diff
@@ -2,19 +2,34 @@
   <a-row :gutter="24">
     <div v-if="isLoaded">
       <a-col :md="24" :lg="7">
-        <a-card :bordered="false" >
+        <a-card :bordered="true" >
```

Comment:
```suggestion
        <a-card :bordered="false" >
```

Reply:
Sem a borda fica ruim de visualizar o que é bloco ou não, acredito que no quesito de visualização faz mais sentido ter a borda

Reply:
Mas isto é desing decision, por padrão o Ant Desing não usa bordas em cards (inclusive no Antdv Pro isto é bem perceptível https://iczer.gitee.io/vue-antd-pro/#/dashboard/workplace) porque os designers da Ant decidiram desta forma.

Claro que poderemos customizar a o tema, mas isto deve ser feito no SCSS, não acho que faça sentido você forçar bordered aqui.
Pelo que entendi você quer sempre manter a borda, mas neste caso isto deve ser feito na folha de estilo pra se perpetuar em todos os cards, faz sentido?

Reply:
Evitar aplicar desing decision em views/components (vide RFC https://github.com/ecomclub/ecomplus-admin/issues/60), isto vai facilitar manter produtividade e respeitar um styleguide, entendeu?

Reply:
Além disto, as coisas ficam pequenas na sua tela...

Reply:
pronto